spec/features/root_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe "check_root_path", type: :feature do
+  before(:all) do
+    FactoryGirl.create(:card).update_column(:review_date, Date.current)
+  end
+
+  before(:each) do
+    visit root_path
+  end
+
+  it "simple visit" do
+    expect(page).to have_content I18n.t('activerecord.attributes.card.translated_text')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.